constructorio-client/src/main/java/io/constructor/client/models/QuizResult.java

```diff
@@ -1,35 +1,79 @@
 package io.constructor.client.models;
 
 import com.google.gson.annotations.SerializedName;
-import java.util.Map;
+import java.util.List;
 
+/** Constructor.io Quiz Result ... uses Gson/Reflection to load data in */
 public class QuizResult {
 
-    @SerializedName("filter_expression")
-    private Map<String, Object> filterExpression;
+    @SerializedName("facets")
+    List<FilterFacet> facets;
```

Comment:
Wonder if we should create a base class and extend it since the results have pretty much the same format across search/browse/rec/quiz except for some minor things.

Reply:
Added a Base Results Response.